core-bundle/src/Resources/contao/library/Contao/Search.php

```diff
@@ -11,6 +11,7 @@
 namespace Contao;
 
 use Contao\Database\Result;
+use Doctrine\DBAL\Driver\Connection;
```

Comment:
```suggestion
use Doctrine\DBAL\Connection;
```

Or is there a specific reason for `Doctrine\DBAL\Driver\Connection` in this case? (We usually use `Doctrine\DBAL\Connection` everywhere else.)

Reply:
We use the interface in the `DefaultIndexer`.

Reply:
I see, may be that's an oversight too there.